exampleProjects/touch/src/main.cpp

```diff
@@ -0,0 +1,127 @@
+//======================== setup pins =======================
+constexpr int TFT_CS { 14 };
+constexpr int TFT_DC { 27 };
+constexpr int pico_SDA { 25 };
+constexpr int pico_SCL { 26 };
+
+//========================= setup screen =======================
+#include <Adafruit_GFX.h>     // Core graphics library
+#include <Adafruit_ILI9341.h> // Hardware-specific library for ST7789
+#include <Adafruit_FT6206.h>
+#include <SPI.h>
+
+Adafruit_ILI9341 tft { TFT_CS, TFT_DC };
+Adafruit_FT6206  ctp;
+
+constexpr int RECT_COUNT    = 16;
+int           SCREEN_HEIGHT = -1;
```

Comment:
For a later PR we should likely be putting a .clang-format file as I don't think this looks like the rest of the formatting we have

Reply:
ah yep, I'll remove the formatting in the next pass -- I was indeed using my own clang-format haha